charts/dm-store/values.yaml

```diff
@@ -1,5 +1,5 @@
 java:
-  image: 'https://hmcts.azurecr.io/hmcts/dm-store:latest'
+  image: 'hmctspublic.azurecr.io/ccd/dm-store:latest'
```

Comment:
docker-compose should also be updated
https://github.com/hmcts/document-management-store-app/blob/master/docker-compose.yml#L7